agent-runner-common/src/main/kotlin/com/epam/drill/test/agent/runner/AgentConfiguration.kt

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.epam.drill.agent.runner
+package com.epam.drill.test.agent.runner
```

Comment:
`.test.` substring in package name / path to class makes it hard to setup ignore rules of test files for Sonar
can we fix it later?